doc/script_commands.txt

```diff
@@ -5317,6 +5317,27 @@ Check getitem2() to understand its expanded parameters.
 
 ---------------------------------------
 
+*delitemidx(<index>{, <amount>{, <account id>}})
+
+This command will remove the item at the given inventory index.
+
+If amount is not specified, it will remove all of the item.
+
+See getinventorylist() for an explanation on retrieving an inventory index.
+
+It is a good idea to check if the player actually has an item at the
+specified index. After deleting an item at an index, that index can
+remain empty until the player re-logs. If you try to delete an item at
+an invalid index or an index with no item, the script will be terminated
+with an error.
```

Comment:
`It is a good idea to check if the player actually has an item at the specified index.`
just say use getiventorylist ...
`After deleting an item at an index, that index can remain empty until the player re-logs.`
and re-use getinventorylist every use before using delitemidx ...

```
This command will remove an item at the given inventory index. Unlike the
'delitem()' counterpart, It doesn't check invalid Item ID, make it useful to remove
invalid item ID in player's inventory.

If <amount> is not specified, this will remove all of the items at the specify index.
Note that items with 'ForceSerial' flag, that not yet 'mergeitem()' will only remove
items at the given index.

The only way to get inventory index is by using 'getinventorylist()'. After deleting
an item at given index, that index can remain empty until the player relog, so have
to recall 'getinventorylist()' again. If you try to delete an item on invalid index, the
script will be terminate with an error.

This command is also useful to remove rental/bound items because 'delitem()'
does not discriminate at choosing which item to remove.
```